app.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python3
+from aws_cdk import App, Environment
+
+from stacks.odincal_stack import OdincalStack
+
+env_EU = Environment(account="991049544436", region="eu-north-1")
+
+app = App()
+
+OdincalStack(
+    app,
+    "OdinSMROdincalStack",
+    level0_queue_arn="OdimSMRLevel0ACFilesQueue",
```

Comment:
arn or name?

Reply:
See above!

Reply:
I changed this to a more obvious placeholder name.